SPECS/python-kubernetes/python-kubernetes.spec

```diff
@@ -79,6 +77,9 @@ cp -pr kubernetes/e2e_test %{buildroot}%{python3_sitelib}/%{library}/
 %{python3_sitelib}/%{library}/e2e_test
 
 %changelog
+* Mon Feb 07 2022 Nicolas Guibourge <nicolasg@microsoft.com> - 21.7.0-1
```

Comment:
It won't let me comment on the relevant line, so I'll comment here. Can you remove the BR on `python3-xml`? It's no longer necessary, as that package was folded into the `python3-libs` subpackage.

Reply:
done